### ISSUE TITLE
vertical_parameter_alignment_on_call should be more flexible if there's a multiline literal

### ISSUE THREAD

**Issue by marcelofabri** [realm/SwiftLint#1643]
```swift
// probably shouldn't trigger
let response = Response(statusCode: 401, data: [
    "error": "invalid_grant",
    "error_description": "The access token provided has expired."
  ], error: NSError(domain: "MyDomain", code: 3213, userInfo: nil))
```

Similar to #1630
